pkg/apis/core/v1beta1/helper/helper.go

```diff
@@ -437,40 +438,15 @@ func ShootMachineImageVersionExists(constraint gardencorev1beta1.MachineImage, i
 	return false, 0
 }
 
-func toExpirableVersions(versions []gardencorev1beta1.MachineImageVersion) []gardencorev1beta1.ExpirableVersion {
+// ToExpirableVersions returns the expirable versions from the given machine image versions
```

Comment:
Similar changes at other places, please end the docstring with full stops.
```suggestion
// ToExpirableVersions returns the expirable versions from the given machine image versions.
```